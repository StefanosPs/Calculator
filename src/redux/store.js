import {createStore, applyMiddleware } from 'redux';


import rootReducer from './root-reducer';

// import logger from 'redux-logger';
// const middlewares = [logger];

// const store = createStore(rootReducer, applyMiddleware(...middlewares));

const store = createStore(rootReducer);

export default store;