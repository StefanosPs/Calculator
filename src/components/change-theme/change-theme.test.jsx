import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from "react-redux";
import store from  '../../redux/store';
import ChangeTheme from './change-theme';
 

it('renders  ChangeTheme', () => {
    const changeTheme = renderer.create(<Provider store={store}><ChangeTheme /></Provider>).toJSON();
    expect(changeTheme).toMatchSnapshot();
});