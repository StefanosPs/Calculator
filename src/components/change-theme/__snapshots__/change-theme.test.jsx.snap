// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  ChangeTheme 1`] = `
<div
  className="switch float-right"
>
  <h6>
    Change Theme
  </h6>
  <label
    className="switch-label"
  >
    <input
      onClick={[Function]}
      type="checkbox"
    />
    <span
      className="slider round"
    />
  </label>
</div>
`;
